### ISSUE TITLE
Theorems to simplify expressions

### ISSUE THREAD

**Issue by jonaprieto** [jonaprieto/agda-metis#19]
Simplify, canonicalize and clausify are inference rules that at some point apply different theorems to simplify the expression. This intends to be that list.

1. All formulas are in their NNF form.

**Comment by jonaprieto** [comment]
```
(* ------------------------------------------------------------------------- *)
(* simplify = isTautology + expandAbbrevs + removeSym                        *)
(* ------------------------------------------------------------------------- *)

val simplify : Thm.thm -> Thm.thm option

(* ------------------------------------------------------------------------- *)
(* simplify = isTautology + expandAbbrevs + removeSym                        *)
(* ------------------------------------------------------------------------- *)

fun simplify th =
    if Thm.isTautology th then NONE
    else
      let
        val th' = th
        val th' = expandAbbrevs th'
        val th' = removeSym th'
      in
        if Thm.equal th th' then SOME th else simplify th'
      end;

```

**Comment by jonaprieto** [comment]
See Simplify module in Rules. Closed by 43108f44fae9f99e544baac7a2fcdaf83ec73bdc.
